src/tests/tracing/eventpipe/bigevent/bigevent.cs

```diff
@@ -0,0 +1,89 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using Microsoft.Diagnostics.Tracing;
+using Tracing.Tests.Common;
+using Microsoft.Diagnostics.Tracing.Parsers.Clr;
+
+namespace Tracing.Tests.BigEventsValidation
+{
+
+    public sealed class BigEventSource : EventSource
+    {
+        private static string bigString = new String('a', 100 * 1024);
```

Comment:
Ideal tests would be:
- throw for 64Kb + 1
-  get results for 64Kb without throw.

Reply:
thanks for the suggestion. This test is actually testing a runtime behavior where it currently crashes if an event greater than 100KB is written via EventPipe. A test that adds 64KB+1 wouldn't trigger that behavior. It also doesn't throw since that's not an API level limit, it's a limit in the underlying event delivery mechanism.